client/my-sites/upgrades/checkout/thank-you.jsx

```diff
@@ -632,4 +643,15 @@ PurchaseDetail = React.createClass( {
 	}
 } );
 
-module.exports = CheckoutThankYou;
+module.exports = connect(
+	function mapStateToProps( state, props ) {
```

Comment:
We have a linting rule against using defining variables that are never used. We should just skip argument this with an anonymous function `() => {}` or `undefined`. The later would be a better solution if it works (and I _think_ it does, based on the [docs](https://github.com/rackt/react-redux/blob/master/docs/api.md#arguments)).